program/print-numbers-from-n-to-1/print_numbers_from_n_to_1.hs

```diff
@@ -0,0 +1,13 @@
+
+
+repeatNTimes n action =
+ do
+  action
+  repeatNTimes (n-1) action
+
+main = do
+  n <- getLine
+  let num = nth (read n :: Integer)
```

Comment:
`nth` variable not in scope error

![image](https://user-images.githubusercontent.com/35031228/195125368-d7c74442-5a7d-45f2-b32d-8e10eb3caac7.png)